hotsos/core/host_helpers/systemd.py

```diff
@@ -58,11 +58,12 @@ def start_time(self):
         fs = FileSearcher()
         # The following expressions need to take account of control characters
         # that might exist in the output e.g. line can start with '*' or U+25CF
+        # Active: active (running) since Wed 2022-02-09 22:38:17 UTC; 17h ago
         seqdef = SequenceSearchDef(
                     start=SearchDef(r'\S+ ({}.service) -'.format(self.name)),
                     body=SearchDef(r"\s+Active: active \(?\S*\)?\s*since "
                                    r"\S{3} (\d{4}-\d{2}-\d{2} "
-                                   r"\d{2}:\d{2}:\d{2})"),
+                                   r"\d{2}:\d{2}:\d{2} .*);"),
```

Comment:
instead of .* can we use \w+ instead so that (a) it stops at the first non-unicode char and (b) we guarantee it exists (since the followup code will fail if it doesnt).

Reply:
> instead of .* can we use \w+ instead so that (a) it stops at the first non-unicode char and (b) we guarantee it exists (since the followup code will fail if it doesnt).

Replaced `.*` with `[\w\+:]+` since timezone strings can contain `+` and `:` on top of word characters.